app/tests/spec/views/base.js

```diff
@@ -349,6 +349,18 @@ define(function (require, exports, module) {
       });
     });
 
+    describe('setHTML', function () {
+      beforeEach(function () {
+        $('#container').html('<div id="stage">stage content</div>');
+        view.setHTML('<div id="stage-child">stage child content</div>');
```

Comment:
Should we have another test that injects an element or have I strayed into the realms of testing that jQuery is working with that suggestion?